### PR TITLE
New package: unigen

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
Not sure what I did wrong here - see `style and docs / validate (pull_request)` stage:

```
Minimum required versions: 3.6
Incompatible versions:     2
Target versions not met:   2.6-, 3.5-
Error: Process completed with exit code 1.
```

or what is wrong with line 13 (which is part of the description).

**Comment by tldahlgren** [comment]
> Not sure what I did wrong here - see `style and docs / validate (pull_request)` stage:
> 
> ```
> Minimum required versions: 3.6
> Incompatible versions:     2
> Target versions not met:   2.6-, 3.5-
> Error: Process completed with exit code 1.
> ```
> 
> or what is wrong with line 13 (which is part of the description).

So f-strings are a Python 3.6 thing, which is a complain in the 3.5 unit tests.  Just a guess but could the special character in line 12 be the reason behind the line 13 issue?

**Comment by iarspider** [comment]
@tldahlgren I was also suspecting accented character to be the reason, didn't think of f-strings.

File: var/spack/repos/builtin/packages/unigen/package.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    """The United Generators project was launched by the Virtual Institute 146
+    VI-SIM in September 2005 following a proposal of Herbert Strobele.
+    The goal was to facilitate comparison between various models (see below)
+    and/or various experiments (HADES, FOPI, CERES, NA49, CBM). The package
+    at present allows to convert output of various event generators to a
+    generic root format."""
+
+    homepage = "https://www.gsi.de/work/wissenschaftliche_netzwerke/helmholtz_virtuelle_institute/unigen.htm"
+    url      = "https://github.com/FairRootGroup/UniGen/archive/v2.3.tar.gz"
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/unigen/package.py:20]:
```suggestion

    maintainers = ['iarspider']
    
```

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/unigen/package.py:20]: Mind adding yourself as the Spack package maintainer?

Comment (iarspider) [reply var/spack/repos/builtin/packages/unigen/package.py:20]: No.

Comment (iarspider) [reply var/spack/repos/builtin/packages/unigen/package.py:20]: I mean, I don't want to add myself as a maintainer - not now.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/unigen/package.py:20]: Thanks for the clarification!

**Review by tldahlgren — APPROVED** [review]
LGTM

**Comment by tldahlgren** [comment]
Thank you for your contribution!

**Pull request merged**